merge_map/solution/merge_map.erl

```diff
@@ -0,0 +1,14 @@
+-module(merge_map).
+
+-export([merge/2]).
+
+merge(Map1, Map2) ->
```

Comment:
If maps:put is allowed, why not just:

```erlang
merge(Map1, Map2) ->
    maps:fold(fun(Key, V1, Map3) ->
                      maps:put(Key, V1, Map3)
              end, Map1, Map2).
``` 
Or, better:
```erlang
merge(Map1, Map2) -> maps:fold(fun maps:put/3, Map1, Map2).
```

Reply:
Thank you! I'll change that.

Reply:
[done :)](https://github.com/lambdaclass/erlings/pull/26/commits/1c408cd55fbeaf47f8f6603a9e7f3c99039ba6b5)